gum/valgrind.h

```diff
@@ -21,16 +21,16 @@
    1. Redistributions of source code must retain the above copyright
       notice, this list of conditions and the following disclaimer.
 
-   2. The origin of this software must not be misrepresented; you must 
-      not claim that you wrote the original software.  If you use this 
-      software in a product, an acknowledgment in the product 
+   2. The origin of this software must not be misrepresented; you must
```

Comment:
Let's not make any whitespace changes to this file. It would make future syncing with upstream harder.

Reply:
Fixed.